interface.h

```diff
@@ -64,150 +68,862 @@
 // ----------------------------------------------------------------------------
 // timers
 
+/**
+ * @brief Adds a new timed event. An existing event timer at the provided index
+ *        identifier will be replaced.
+ *
+ * @param index Used to register and identify the timed event. Valid values are
+ *        between 0 and TIMED_EVENT_COUNT - 1
+ * @param ms The timed event interval in milliseconds
+ * @param repeat 0 or 1 value indicating if the timed event should repeat
+ *        indefinitely
+ */
 void add_timed_event(uint8_t index, uint16_t ms, uint8_t repeat);
+
+/**
+ * @brief Stops and removes a previously added timed event at the index
+ *        identifier.
+ * 
+ * @param index Index identifier of the timed event to stop
+ */
 void stop_timed_event(uint8_t index);
+
+/**
+ * @brief Updates a previously registed timed event trigger interval in
+ *        milliseconds
+ * 
+ * @param index Index identifier of the timed event to update
+ * @param ms How often the timed event should be triggered in milliseconds
```

Comment:
change this to `The timed event interval in milliseconds` to keep it consistent with `add_timed_event`

Reply:
i can change it when i update multipass!